compressor/OffsetAndLength.h

```diff
@@ -0,0 +1,15 @@
+class OffsetAndLength{
+  Byte offset;
+  Byte length;
+
+public:
+  OffsetAndLength(){ offset = 0; length = 0;}
+
+  void set(int offs, int len, bool compressed = false);
```

Comment:
I guess you don't need separate set function for offset and length. Only one set for both.